AutomatedTesting/Gem/PythonTests/Atom/atom_utils/material_editor_utils.py

```diff
@@ -241,14 +241,13 @@ def capture_screenshot_blocking(self, filename):
         """
         Capture a screenshot and block the execution until the screenshot has been written to the disk.
         """
-        self.handler = azlmbr.atom.FrameCaptureNotificationBusHandler()
-        self.handler.connect()
-        self.handler.add_callback("OnCaptureFinished", self.on_screenshot_captured)
-
         self.done = False
         self.capturedScreenshot = False
-        success = azlmbr.atom.FrameCaptureRequestBus(azlmbr.bus.Broadcast, "CaptureScreenshot", filename)
-        if success:
+        frameCaptureId = azlmbr.atom.FrameCaptureRequestBus(azlmbr.bus.Broadcast, "CaptureScreenshot", filename)
+        if frameCaptureId != -1:
+            self.handler = azlmbr.atom.FrameCaptureNotificationBusHandler()
+            self.handler.connect()
+            self.handler.add_callback("OnCaptureFinished", self.on_screenshot_captured)
```

Comment:
Since the bus now has an ID, don't we need to use frameCaptureId in the connect function call or something like that?

Reply:
Yes, good catch.

Reply:
Took another look, I don't think this code is used anywhere.

Reply:
Updated.